### DISCUSSION TITLE
@aws-sdk/core: No matching version found for @aws-sdk/core@3.362.0.

### DISCUSSION THREAD

**Discussion by ericxinzhang** [aws/aws-sdk-js-v3#5765] [Q&A]
As the [Lambda runtime](https://docs.aws.amazon.com/sdkref/latest/guide/version-support-matrix.html) says the OOTB SDK version for node 18 is 3.362.0, I am trying to install `@aws-sdk/core@3.362.0` but got following error:

```
npm verb type version
npm verb stack @aws-sdk/core: No matching version found for @aws-sdk/core@3.362.0.
```

Any advice?

**Comment by jedwards1211** [comment]
This keeps happening periodically, either their automated publishing doesn't publish packages in the correct order or NPM's CDN is getting slower to update...
